vault/wrapping.go

```diff
@@ -184,6 +184,11 @@ DONELISTHANDLING:
 		resp.WrapInfo.WrappedAccessor = resp.Auth.Accessor
 	}
 
+	// Make secret_id_accessor available as a WrappedAccessor.
+	if secretIdAccessor, ok := resp.Data["secret_id_accessor"]; ok {
```

Comment:
would it make sense to be a bit defensive and make this check `ok && resp.Auth == nil` , to be certain we don't overwrite token accessors? ( Unlikely as it is to be an actual problem )

Reply:
That sounds sensible to me. Added it.